corehq/apps/export/utils.py

```diff
@@ -0,0 +1,8 @@
+from .const import TRANSFORM_FUNCTIONS
+from .exceptions import ExportInvalidTransform
+
+
+def is_valid_transform(value):
+    for transform in value:
+        if transform not in TRANSFORM_FUNCTIONS:
+            raise ExportInvalidTransform('{} is not a valid transform'.format(value))
```

Comment:
oh interesting, that's what i originally had when it was a StringProperty (for a single transform). didn't think it was smart enough to handle a ListProperty. i'll mark to update